.github/workflows/release.yml

```diff
@@ -891,7 +804,7 @@ jobs:
   release:
     name: Create release
     runs-on: ubuntu-latest
-    if: github.ref != 'refs/heads/main'
+    if: startsWith(github.ref, 'refs/tags/')
```

Comment:
This makes it a little easier to run the release build for a one-off branches, because this will _only_ run for tags.